src/renderer/Components/RecevingPackage/index.jsx

```diff
@@ -45,9 +59,54 @@ function ReceivingPackage() {
     navigate("/");
   };
 
-  const handleReceiveResult = (event) => {
+  const handleReceivePackage = async (event) => {
     event.preventDefault();
-    setIsModalOpen(true);
+    try {
+      const serialNumber = inputNumbers.join("");
+      const response = await axios.get(
+        `${import.meta.env.VITE_SERVER_URL}/packages/${serialNumber}`,
+      );
+      const orderList = response.data.existPackage.orders;
+      // TODO: 콘솔로그를 일렉트론의 유틸함수와 연결해주기
+      async function processActions() {
```

Comment:
저희 프로젝트에서 함수 정의할때 선언식 대신 표현식 형태로 하기로 했으므로 다음과 같이 수정부탁드립니다. 😄

```suggestion
      const processActions = async () => {
```

Reply:
앗 무심코 익숙한 선언문 형식을 써버렸네요... 매의눈으로 체크해주셔서 감사합니다!!😊